src/controllers/dynakube/dtclient_reconciler.go

```diff
@@ -158,6 +159,8 @@ func (r *DynatraceClientReconciler) Reconcile(ctx context.Context, instance *dyn
 		})
 	}
 
+	tokens[0].OptionalScopes = append(tokens[0].Scopes, dtclient.TokenScopeActiveGateTokenCreate)
```

Comment:
`tokens` should be a `map[TokenType]tokenConfig` instead of `[]tokenConfig` to avoid relying on order in the slice

```
type TokenType int64

const (
	ApiToken TokenType = iota
	PaasToken 
)
```

Reply:
Good suggestion, thanks!